### ISSUE TITLE
[VPN] Gateway & connection

### ISSUE THREAD

**Issue by MrLuje** [FlexibleEngineCloud/terraform-provider-flexibleengine#685]
Hi,

AFAIK there is no terraform resources yet to create a fully fonctionnal VPN (VPN Gateway + connections + route tables, etc)
Is there anything planned on the roadmap ?

Thanks

**Comment by ShiChangkuo** [comment]
can you do such work by console?

**Comment by MrLuje** [comment]
> can you do such work by console?

Yes, through Network Console > Virtual Private Network... but not sure if there is an API for it

**Comment by ShiChangkuo** [comment]
I found the VPN service, but the API is missing...

**Comment by Pseudodevincent** [comment]
Hello, unfortunately, there is no API yet on VPN service... A new version of VPN service which will include API support is in the the roadmap for Q4 2022 but I don't have any details yet.
